### ISSUE TITLE
The table in the "Server Activity" tab is a keyboard trap

### ISSUE THREAD

**Issue by erikjv** [owncloud/client#11789]
Inside this table, you can navigate with the arrow keys, but also with tab. The problem is, that you cannot escape using tab: when the last cell is selected, tab will bring focus to the first cell.
